src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -81,7 +81,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $routes = $this->router->getRouteCollection();
 
         if ($name) {
-            if (!($route = $routes->get($name)) && $matchingRoutes = $this->findRouteNameContaining($name, $routes)) {
+            $route = $routes->get($name);
+            $matchingRoutes = $this->findRouteNameContaining($name, $routes);
+
+            if (!$input->isInteractive() && !$route && \count($matchingRoutes) > 1) {
+                $foundRoutes = $this->findRouteContaining($name, $routes);
+                $helper->describe($io, $foundRoutes, [
```

Comment:
The variable is not necessary.
```suggestion
                $helper->describe($io, $this->findRouteContaining($name, $routes), [
```